soroban-spec/src/gen/ts/project_template/src/invoke.ts

```diff
@@ -67,8 +69,25 @@ export async function invoke({ method, args = [], fee = 100 }: InvokeArgs): Prom
     .build()
 
   const simulated = await Server.simulateTransaction(tx)
+  
+  const auths = simulated.results?.[0]?.auth
+
+  // is it possible for `auths` to be present but empty? Probably not, but let's be safe.
+  if (auths && auths.length > 0) {
```

Comment:
```suggestion
  if (auths?.length > 0) {
```

You can do this, no?

Reply:
yes, thanks, pushed